Source/Kesmai.Server/Game/Items/Equipment/Weapons/Staves/RamStaff.cs

```diff
@@ -0,0 +1,62 @@
+using System.Collections.Generic;
+using System.IO;
+using Kesmai.Server.Game;
+using Kesmai.Server.Network;
+
+namespace Kesmai.Server.Items
+{
+	public partial class RamStaff : Staff
+	{
+        /// <inheritdoc />
+        public override int LabelNumber => 6000088;
+
+		/// <inheritdoc />
+		public override uint BasePrice => 800;
+
+		/// <inheritdoc />
+		public override int Weight => 1400;
+
+        /// <inheritdoc />
+        public override ShieldPenetration Penetration => ShieldPenetration.Medium;
+
+		/// <inheritdoc />
+		public override int Category => 1;
+
+		/// <inheritdoc />
+		public override int MinimumDamage => 1;
+
+		/// <inheritdoc />
+		public override int MaximumDamage => 8;
+
+        /// <inheritdoc />
+        public override bool CanBind => true;
+
+		/// <inheritdoc />
+		public override int BaseArmorBonus => 4;
+
+		/// <inheritdoc />
+		public override int BaseAttackBonus => 2;
+
+        /// <inheritdoc />
+        public override int ProjectileProtection  => 1;
+
+		/// <inheritdoc />
+		public override WeaponFlags Flags => WeaponFlags.TwoHanded | WeaponFlags.Bashing | WeaponFlags.Lawful
+		
+		/// <summary>
+		/// Initializes a new instance of the <see cref="BlackStaff"/> class.
+		/// </summary>
+		public BlackStaff() : base(901)
```

Comment:
Public RamStaff() here

Reply:
oof good catch - removed that and added.

Reply:
#1545 hasn't gone through yet, and changes your ID to 902
(To keep the non poison and poison daggers together.)